src/services/api-service.ts

```diff
@@ -44,6 +45,25 @@ class ApiService {
     this.logger.structuredLogError = structuredLogError
   }
 
+  logInfo(description: string, structuredLogInfo?: StructuredLogInfo) {
```

Comment:
Nice utility methods

Reply:
@alexanderattar @ACC01ADE So we had an issue with calling the `this.logger.structuredLog` function because of the `this` context. I believe we are good, but im just making sure.